public/Dockerfile

```diff
@@ -1,3 +1,8 @@
 FROM nginx:alpine
 
-COPY ./ /usr/share/nginx/html/
\ No newline at end of file
+COPY ./nginx.conf /etc/nginx/nginx.conf
+COPY ./ /usr/share/nginx/html/
+# Delete the nginx.conf file so that we don't serve our config via nginx
+# This is because I don't know an easier way to COPY all files _except_ that one.
+RUN rm /usr/share/nginx/html/nginx.conf
```

Comment:
Could we move this file over to a `.docker` folder instead of having it in the `public` folder?

Reply:
Maybe? I was pretty unclear on how else to end up with the file inside of `build/`, which is the docker context for the image build. But it occurs to me now that the docker build context can be whatever we want it to be, so I can definitely move that conf file so we don't have to delete it later during the image build. Is there some special handling with `.docker` and create-react-app? I couldn't immediately find and docs on that.

Reply:
That's a fair point and I think we can leave it as this is the easiest. In other setup docs I've read they just tell you to copy the build into the folder you want... and that's basically the same amount of work/coding as this is.

Reply:
Ok, I just tried a quick test, and I don't see an out-of-the-box way to have other stuff end up in that `build/` dir. I also looked at what it would take to just change the docker build context, and decided that's not worth it now, either, because `gcloud run deploy` doesn't support separate arguments for the source Dockerfile and the context. We can revisit that later, though, if we want to change to build images directly instead of through `gcloud run deploy`, but :shrug: